administracion/views.py

```diff
@@ -38,3 +40,14 @@ def admin_users_create(request):
         if forma.is_valid():
             forma.save()
             return redirect('administracion:users')
+    else:
+        forma = FormaCreacionUsuario()
+    return render(request, 'crear_usuario.html', {'form': forma})
+
+
+@login_required
+@user_passes_test(User, is_administrador)
```

Comment:
there's no need to pass the first parameter to this function. check what is above.